google/datalab/contrib/bigquery/operators/_bq_load_operator.py

```diff
@@ -67,3 +65,7 @@ def execute(self, context):
       raise Exception('Load failed: %s' % str(job.fatal_error))
     elif job.errors:
       raise Exception('Load completed with errors: %s' % str(job.errors))
+
+    return {
+      'result': job.result()
```

Comment:
Maybe the job itself.
And the resulting table in addition.

Reply:
See concern above about using xcom for communicating non-primitive types. 

In general, we can go about this in two ways - using xcom to communicate all information, and using the input spec to statically get all the information we need. Agreed that the former gives us more flexibility. However, it seems like something we could do later, once this is more stable. 

Filed P1 https://github.com/googledatalab/pydatalab/issues/569